New changes SQL/XT Voidzone add.sql

```diff
@@ -0,0 +1 @@
+UPDATE creature_template SET ScriptName ='npc_xt_void_zone' WHERE entry = 34001;
```

Comment:
delete this file, it contains the same data has sql/updates/world/2016_06_25_00_world_335.sql (btw remove 335 there)